### ISSUE TITLE
FRP: Add default `settings` pages

### ISSUE THREAD

**Issue by Majorfi** [Major-Eth/yUITemplate#52]
### What is your proposal 🚀
Add a default pages used to be able to overwrite some environment variables and/or specific endpoints.

### What does this solve ? 🛠
The motivation is simple: make our products resilients. What if, tomorrow, IPFS is still there with our project, but the API is down ? Now way to use the build for some projects. Being able to change the RPC, to change yDaemon endpoints etc. is a big win to avoid that and to allow users to use their  localhost endpoints

### More 🔎
*N/A*

**Comment by ai-slave** [comment]
Just checking if work that has been done on this task doesn't interfere with these:
<img width="997" alt="image" src="https://user-images.githubusercontent.com/6696080/202158860-dfbf6a4c-577c-4211-b098-106acd0ea079.png">

**Comment by 0xBasically** [comment]
Yes please this would be great to add, some of our sites have this while some do not
